### ISSUE TITLE
[Improvement/Suggestion] Reset stats button

### ISSUE THREAD

**Issue by veamorim** [favll/pogom#268]
## Checklist
 - [x] I have read the [FAQ](https://github.com/favll/pogom/blob/master/README.md#faq)
 - [x] I have checked if this issue already exists (both open and closed issues)
 - [x] I am using the most recent version of pogom from the master branch
 - [ ] If this is an installation issue, I am posting my OS, OS version, python version, pip version
 - [ ] I'm posting the complete log
 - [x] I'm using a descriptive title for this issue

## My Issue
Basically, a button on stats screen to reset the list.
It is good because sometimes I want to know if a pokemon is going to show up in a specific area, and I have to check and see how many I already found, and then check if it is the same after a few hours.
...

**Comment by nborrmann** [comment]
>It is good because sometimes I want to know if a pokemon is going to show up in a specific area, and I have to check and see how many I already found, and then check if it is the same after a few hours.
I don't see the problem here tbh.

If you want to reset stats, delete the "pogom.db" file (or rename it).
